src/components/ForgotUserName/ForgotUserNameForm.js

```diff
@@ -93,23 +105,24 @@ class ForgotUserName extends Component {
               >
                 <FormattedMessage id="stripes-core.button.continue" />
               </Button>
-              <div
-                className={formStyles.authErrorsWrapper}
-                data-test-errors
-              >
-                {hasErrorsContainer &&
-                  <Row center="xs">
-                    <Col xs={12}>
+              <Row center="xs">
+                <Col xs={12}>
+                  <div
+                    className={formStyles.authErrorsWrapper}
+                    data-test-errors
+                  >
+                    {hasErrorsContainer &&
                       <AuthErrorsContainer
```

Comment:
Please consider wrapping it with parenthesis as it is a multiline JSX expression.